Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -183,45 +281,45 @@ public double pidGet() {
 		return average(leftEnc.getDistance(), rightEnc.getDistance());
```

Comment:
javadocs for this method???

Reply:
Implemented method for overriding PIDSource, I think it's unnatural to add javadocs for methods that aren't yours :)